ena_upload/ena_upload.py

```diff
@@ -101,23 +102,26 @@ def check_columns(df, schema, action, dev, auto_action):
                     for index, row in df.iterrows():
                         remote_present = np.nan
                         try:
-                            remote_present = str(identify_action(
-                                schema, str(df['alias'][index]), dev)).upper()
+                            remote_present = remote_check(
+                                schema, str(df['alias'][index]), dev)
 
                         except Exception as e:
                             print(e)
                             print(
                                 f"Something went wrong with detecting the ENA object {df['alias'][index]} on the servers of ENA. This object will be skipped.")
-                        if remote_present == np.nan:
-                            df.at[index, header] = np.nan
-                        elif remote_present and action == 'MODIFY':
+                        if remote_present and action == 'MODIFY':
                             df.at[index, header] = action
                             print(
-                                f"\t'{df['alias'][index]}' gets '{remote_present}' as action in the status column")
+                                f"\t'{df['alias'][index]}' gets '{action}' as action in the status column")
                         elif not remote_present and action in ['ADD', 'CANCEL', 'RELEASE']:
                             df.at[index, header] = action
                             print(
-                                f"\t'{df['alias'][index]}' gets '{remote_present}' as action in the status column")
+                                f"\t'{df['alias'][index]}' gets '{action}' as action in the status column")
+                        else:
```

Comment:
should that not be an explicit check for np.nan as above?

Reply:
the behavior of `remote_present` changed, with following logic: 
- if the object exists on ENA, and the action is modify, that object will be modified
- if the object exists on ENA and the action is add, release or cancel, this will get skipped (no action filled in) because ENA does not let you do that (this is the else statement)
- if the object does not exist than add, release or cancel will be added if that is the action
- if auto_action is not activated, all the previous ones are not checked and the actions in the column are used, if given, if not given the column gets added with for every line the action that is given in the CLI command